src/askContainer/commands/cloneSkill.ts

```diff
@@ -37,14 +38,22 @@ export class CloneSkillCommand extends AbstractCommand<void> {
             skillInfo.data.skillSummary.nameByLocale!);
         const filteredProjectName = Utils.filterNonAlphanumeric(skillName)
         const skillFolderAbsPath = path.join(projectFolder.fsPath, filteredProjectName);
-        
+
         // create skill folder in project path
         if (fs.existsSync(skillFolderAbsPath)) {
-            throw loggableAskError(`Skill folder ${skillFolderAbsPath} already exists.`);
+
+            const errorMessage = `Skill folder ${skillFolderAbsPath} already exists. Would you like to override it?`;
```

Comment:
Message should be `...Would you like to overwrite it?`, right? since on clicking `Yes`, we are overwriting the folder?